src/streamsync/serve.py

```diff
@@ -408,7 +408,7 @@ def print_route_message(run_name: str, port: int, host: str):
     GREEN_TOKEN = "\033[92m"
     END_TOKEN = "\033[0m"
 
-    print(f"{run_name} is available at:{END_TOKEN}{GREEN_TOKEN} http://{host}:{port}{END_TOKEN}")
+    print(f"{run_name} is available at:{END_TOKEN}{GREEN_TOKEN} http://{host}:{port}{END_TOKEN}", flush=True)
```

Comment:
Really curious about the `flush`

Reply:
Thats needed by `ui_test_app` which is determining readiness status of the sub process based on this message. Sometimes program was stuck on waiting for this message when it was buffered.